types/errors.go

```diff
@@ -286,7 +287,57 @@ func (err *sdkError) QueryResult() abci.ResponseQuery {
 	}
 }
 
-// nolint
+//----------------------------------------
+// REST error utilities
```

Comment:
Hmm I don't think this is the best approach, it makes all of this code very dependent on the upstream `Error()` function - maybe we should change the upstream instead?

Reply:
updated on upstream 👍